enacts/enactstozarr.py

```diff
@@ -80,5 +85,6 @@ def convert(variable):
     print(f"conversion for {variable} complete.")
     return zarr
 
-for i in CONFIG['vars']:
-    convert(i)
+time_res="dekadal"
```

Comment:
This shouldn't be hard-coded, should it? Did you mean to iterate over all the resolutions?

Reply:
I have written this keeping [this](https://github.com/iridl/python-maprooms/pull/277/files#r1091048771) in mind. But maybe that doesn't apply here?

Reply:
Drew's comment on the conversation you linked to was

> someone could just comment out variables they wouldn't want for now.

That approach could be generalized to multiple temporal resolutions like this:
```
temporal_resolutions = [
    "daily",
    "dekadal",
]
for t in temporal_resolutions:
    for v in CONFIG['vars']:
        convert(v, t)
```